package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "to-do_list",
+  "version": "1.0.0",
+  "description": "In this project, I had to build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed in the lesson. This simple web page will be built using webpack and served by a webpack dev server.",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve --open"
```

Comment:
✅ Webpack dev server is set up and running with no errors 👍🏻.

Reply:
:100: %